### ISSUE TITLE
Unable to update within web UI when using under subfolder

### ISSUE THREAD

**Issue by UNOPARATOR** [linuxserver/docker-nextcloud#138]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

If you are new to Docker or this application our issue tracker is **ONLY** used for reporting bugs or requesting features. Please use [our discord server](https://discord.gg/YWrKVTn) for general support.

Using this docker image in UNRAID v6.8.3 with the latest tag which is the same as 18.0.3-ls73 at this time. 
Also linuxserver/letsencrypt is installed in UNRAID and NextCloud is configured to run as a subfolder *(/nextcloud)* according to your sample.conf file.
After updating the container and I tried to upgrade the NextCloud via the web ui from 17.0.1 to 18.0.3.

------------------------------

## Expected Behavior
Updater should work.

## Current Behavior
The updater fails because the following html element is not pointing to the subfolder:
```html
<input type="hidden" id="updater-endpoint" value="/updater/index.php">
```
instead of:
```html
<input type="hidden" id="updater-endpoint" value="/nextcloud/updater/index.php">
```
Making this change from the Developer Tools of the web browser, I was able to start the update.
But in several steps, a similar error showed up.
I was able to continue the update process by going back to the Settings --> Administration --> Overview page and pressing the "Open Updater" button again (sometimes I needed to wait a bit, for long taking steps to be completed in order see the page; ie. downloading, backing up, deleting old files...) and pressing the "Continue Update" button.

## Steps to Reproduce

1. Have a previous version of NextCloud installed and configured to run a subfolder *(/nextcloud)*
2. Open the NextCloud web ui, login as an administrator
3. Go to Settings --> Administration --> Overview and see that there is an update waiting
4. Press the "Open Updater" button and try to proceed with the update

## Environment
**OS:**      UNRAID v6.8.3
**CPU architecture:** x86_64
**How docker service was installed:**      

## Command used to create docker container (run/create/compose/screenshot)

![image](https://user-images.githubusercontent.com/1214176/78188231-8e44ea00-7478-11ea-8f2b-2a00b647796f.png)

**Comment by CHBMB** [comment]
Are you sure you don't mean sub**folder** not subdomain?

As in I think you mean `server.com/nextcloud` whereas subdomain means `nextcloud.server.com`

Running as a subfolder (which is what I *think* you're doing is not a great way to run Nextcloud to be honest and there have been issues raised by others in Nextcloud.

**Comment by UNOPARATOR** [comment]
> Are you sure you don't mean sub**folder** not subdomain?
> 
> As in I think you mean `server.com/nextcloud` whereas subdomain means `nextcloud.server.com`
> 
> Running as a subfolder (which is what I _think_ you're doing is not a great way to run Nextcloud to be honest and there have been issues raised by others in Nextcloud.

Yes you are right, in my first sentence I typed subfolder but in the following *(and the topic)* I mistyped as subdomain, sorry.
I gave up on using as subfolder in the last week or two.
Since I also use letsencrypt by linuxserver and I saw the option to use both subdomain & subfolder, I chose the latter *(didn't want to use up 1 out of 5 free duckdns subdomains)*. Also there were precautions about this issue which is why I wanted to raise it.
For example, I also use codimd by linuxserver but I had to use it as a subdomain since letsencrypt didn't have a subfolder sample config in the first place *(although I tried to create one myself but failed so gave up)*.
